airbyte-integrations/connectors/destination-snowflake/src/test/java/io/airbyte/integrations/destination/snowflake/SnowflakeSqlOperationsTest.java

```diff
@@ -79,4 +79,26 @@ public void testCreateSchemaIfNotExists(final String message, final boolean shou
     }
   }
 
+  @ParameterizedTest
+  @CsvSource({"TEST,false", "but current role has no privileges on it,true"})
+  public void testCreateTableIfNotExists(final String message, final boolean shouldCapture) {
+    final JdbcDatabase db = Mockito.mock(JdbcDatabase.class);
+    final var schemaName = "foo";
+    final var tableName = "bar";
```

Comment:
Minor nit: since we know these values in the `createTableIfNotExists` takes in `String` values to also just have the test reflect that. It shouldn't be necessary that the tests are using type inferences when the method defines the type explicitly